src/email.rs

```diff
@@ -91,6 +91,32 @@ or go to https://{domain}/me/pending-invites to manage all of your crate ownersh
         self.send(email, subject, &body)
     }
 
+    /// Attempts to send a notification that a new crate may be typosquatting another crate.
+    pub fn send_possible_typosquat_notification(
+        &self,
+        email: &str,
+        crate_name: &str,
+        squats: &[typomania::checks::Squat],
+    ) -> AppResult<()> {
+        let subject = "Possible typosquatting in new crate";
+        let body = format!(
+            "New crate {crate_name} may be typosquatting one or more other crates.\n
+Visit https://{domain}/crates/{crate_name} to see the offending crate.\n
+\n
+Specific squat checks that triggered:\n
+\n
+- {squats}\n",
+            domain = crate::config::domain_name(),
+            squats = squats
+                .iter()
+                .map(|squat| format!("{squat}"))
```

Comment:
```suggestion
                .map(ToString::to_string)
```

I think something like that should work too, right?

Reply:
```suggestion
                .map(|squat| format!("- {squat}\n"))
```

This may be an alternative which is a bit more straight-forward compared to the `- {squats}\n` approach above 😅

Reply:
one more thought that just passed my mind: we are linking to `https://{domain}/crates/{crate_name}` above, but I guess it would be nice to also link to the squatted crates in this list?

Reply:
Agreed; done.